meta-sparrow/recipes-kernel/linux/linux-sparrow/0005-Create-copy-of-devfreq_trace.h.patch

```diff
@@ -1,4 +1,4 @@
-From 7e53114b1f8b917ef9f4fe1e1cf6908526b7ffc7 Mon Sep 17 00:00:00 2001
```

Comment:
It's not clear why these hash values changed for a number of otherwise untouched patch files and only for sparrow.  Was this some devtool artifact?

Reply:
Correct, this is just an artifact from a `devtool finish` iirc.

I can just remove those changes if you want me too?

Reply:
Not necessary -- just wanted to make sure.